src/services/LoggerService.ts

```diff
@@ -243,13 +232,8 @@ export default class LoggerService extends DependencyAwareClass {
    * @param silent If `false`, the metric will also be logged. (default: false)
    */
   metric(descriptor: string, stat: number | string, silent = false) {
-    if (
-      typeof process.env.EPSAGON_TOKEN === 'string'
-      && process.env.EPSAGON_TOKEN !== 'undefined'
-      && typeof process.env.EPSAGON_SERVICE_NAME === 'string'
-      && process.env.EPSAGON_SERVICE_NAME !== 'undefined'
-    ) {
-      Epsagon.label(descriptor, stat);
+    if (process.env.LUMIGO_TRACER_TOKEN) {
```

Comment:
If Lumigo auto-tracing is enabled, am I right in thinking `LUMIGO_TRACER_TOKEN` will *not* be set? Therefore `lumigo.addExecutionTag()` will not be called while auto-tracing?